languages/go/exercises/concept/time-duration/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you'll be working on an class room time schdule for a school. 
+
+You have five tasks, which will all involve scheduling of class periods and intervals.The class duration is 1 hr in all the tasks given blow.
+
+## 1. Parse the Duration of the Class
+
+Implement the `StartTime` function to parse a textual representation of a time duration of the class into the corresponding `time.Duration` format and print the time:
+
+```go
+StartTime("1h")
+// => 1h0m0s
+```
```

Comment:
Parsing a `duration`, perfect! I'd change the story however as `StartTime` is more like a time, not a `duration`.

Reply:
Yeah, maybe `ClassDuration` or something like that?